src/more-infos/more-info-alarm_control_panel.html

```diff
@@ -6,12 +6,12 @@
 <dom-module id='more-info-alarm_control_panel'>
   <template>
       <div class='layout horizontal'>
-          <paper-input label='code' value='[[entered_code]]' pattern='[0-9]*' type='password' on-change='enteredCodeChanged'></paper-input>
+	      <paper-input label='code' input='[[entered_code]]' pattern='[[code_format]]' on-keyup='entered_code_changed' type='password' hidden$='[[!code_input_visible]]' disabled$='[[!code_input_enabled]]'></paper-input>
```

Comment:
Few comments about this line:
 - Please add some enters to make it more readable
 - like you commented yourself, `input` -> `value`
 - if you change `value` attribute to `{{entered_code}}`, the passed in value will be kept in sync with the input box
 - This would remove the need for `on-keyup`